encrypter.cpp

```diff
@@ -0,0 +1,177 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <ios>
+#include <random>
+#include <sstream>
+#include <utility>
+#include <iterator>
+
+using Byte = char;
+
+//=====put_error=====//
+void put_error(const std::string& error_message)
+{
+  std::cout << error_message;
+  exit(1);
+}
+
+//=====shuffle=====//
+void shuffle(std::vector<Byte>& input)
+{
+  std::default_random_engine gen;
+
+  for (size_t i = 0; i < input.size(); ++i)
+  {
+    std::uniform_int_distribution<int> dis (i, input.size() - 1);
+    std::swap(input[i], input[dis(gen)]);
+  } 
+}
+
+//=====write_file=====//
```

Comment:
Better get rig of the name of the function in each comment. Because each function name repeated twice.
Leave just //====================// if you want some separator.